### PR TITLE
Updating branding to rtm 

### PR DESCRIPTION
**Pull request by Anipik**
- git checkout release/5.0
- git merge release/5.0-rc2
https://github.com/dotnet/runtime/commit/d2e8221c4d18df2f01010eb3867f989fe9cfc7b3 causes merge conflicts
- resolve conflicts in favor of rc2
- update branding and push.

Still waiting for tactics to give a go ahead for this one.

### CODE REVIEW

**Review by mmitche — APPROVED** [review]

**Review by safern — APPROVED** [review]

File: src/coreclr/tests/issues.targets
Comment (SamMonoRT) [inline src/coreclr/tests/issues.targets:2721]: Just confirming, the original file is based off RC1/2 or off main ?  (In main this file issues.target, has been moved to https://github.com/dotnet/runtime/tree/master/src/tests )

Comment (Anipik) [reply src/coreclr/tests/issues.targets:2721]: its based off rc2

**Comment by Anipik** [comment]
Update:- git merged again today. (There were no conflicts today)

**Comment by Anipik** [comment]
@ericstj @safern @mmitche can u take another look before i merge this ? i merged the rc2 changes after the latest approval.

**Comment by Anipik** [comment]
```
C:\git\runtime>git diff upstream/release/5.0-rc2
diff --git a/eng/Versions.props b/eng/Versions.props
index 6870068f2fb..9ac70f317ac 100644
--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -6,8 +6,7 @@
     <MajorVersion>5</MajorVersion>
     <MinorVersion>0</MinorVersion>
     <PatchVersion>0</PatchVersion>
-    <PreReleaseVersionLabel>rc</PreReleaseVersionLabel>
-    <PreReleaseVersionIteration>2</PreReleaseVersionIteration>
+    <PreReleaseVersionLabel>rtm</PreReleaseVersionLabel>
     <!-- Set assembly version to align with major and minor version,
          as for the patches and revisions should be manually updated per assembly if it is serviced. -->
     <AssemblyVersion>$(MajorVersion).$(MinorVersion).0.0</AssemblyVersion>

C:\git\runtime>
```

**Review by ViktorHofer — APPROVED** [review]

**Pull request merged**
